lesson_5/Task11.java

```diff
@@ -0,0 +1,19 @@
+package lesson5;
+ //What is the difference between package-private access and protected access?
```

Comment:
I don't see an answer on this question.

Reply:
The private modifier specifies that the member can only be accessed in its own class. The protected modifier specifies that the member can only be accessed within its own package (as with package-private) and, in addition, by a subclass of its class in another package